pkg/wait/wait_for_ready.go

```diff
@@ -110,20 +104,9 @@ func (w *waitForReadyConfig) Wait(name string, timeout time.Duration, msgCallbac
 	}
 }
 
-func addWatchTimeout(opts *v1.ListOptions, timeout time.Duration) {
```

Comment:
should be accessible in `poll_watcher.go` as well (as we moved this in `poll_watcher.go`)